frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -1,224 +1,128 @@
+  const { deployment, externalStorage, type } = backingStorage;
 
-  const validations = [...capacityAndNodesValidate(nodes, enableArbiter)];
   const isNoProvisioner = storageClass.provisioner === NO_PROVISIONER;
+  const formattedCapacity = !isNoProvisioner ? `${OSD_CAPACITY_SIZES[capacity]} TiB` : capacity;
+
+  const hasEncryption = encryption.clusterWide || encryption.storageClass;
+
+  const storagePlatform = externalStorage && getExternalStorage(externalStorage).displayName;
+
+  const encryptionStatus = hasEncryption
+    ? t('ceph-storage-plugin~Enabled')
+    : t('ceph-storage-plugin~Disabled');
 
   const totalCpu = getTotalCpu(nodes);
   const totalMemory = getTotalMemory(nodes);
   const zones = getAllZone(nodes);
 
   return (
-    <dl>
-      <ReviewListTitle text={t('ceph-storage-plugin~Backing storage')} />
-      <ReviewListBody hideIcon>
-        <span>{t('ceph-storage-plugin~StorageClass:')}</span>&nbsp;
-        <span className="text-secondary">{storageClass.name}</span>
-      </ReviewListBody>
-      <ReviewListTitle text={t('ceph-storage-plugin~Capacity and nodes')} />
-      <ReviewListBody hideIcon>
-        <span>{t('ceph-storage-plugin~Requested Cluster Capacity:')}</span>&nbsp;
-        <span className="text-secondary">
-          {isNoProvisioner ? (
-            <>
-              {OSD_CAPACITY_SIZES[capacity]} TiB&nbsp;
-              <TotalCapacityText capacity={capacity} />
-            </>
-          ) : (
-            capacity
-          )}
-        </span>
-      </ReviewListBody>
-      <ReviewListBody noValue={nodes.length < MINIMUM_NODES}>
-        <div>
-          <p>
-            {t('ceph-storage-plugin~{{nodeCount, number}} node', {
-              nodeCount: nodes.length,
-              count: nodes.length,
-            })}{' '}
-            {t('ceph-storage-plugin~selected')}
-          </p>
-          <NodesCard nodes={nodes} />
-        </div>
-      </ReviewListBody>
-      <ReviewListBody
-        validation={validations.includes(ValidationType.MINIMAL) && ValidationType.MINIMAL}
-        noValue={!totalCpu || !totalMemory}
-      >
-        <p>
-          {t('ceph-storage-plugin~Total CPU and memory of {{cpu, number}} CPU and {{memory}}', {
+    <>
+      <ReviewItem title={t('ceph-storage-plugin~Backing storage')}>
+        <ListItem>
+          {t('ceph-storage-plugin~StorageClass: {{name}}', {
+            name: storageClass.name || createLocalVolumeSet.volumeSetName,
+          })}
+        </ListItem>
+        <ListItem>
+          {t('ceph-storage-plugin~Deployment type: {{deployment}}', {
+            deployment,
+          })}
+        </ListItem>
+        {type === BackingStorageType.EXTERNAL && (
+          <ListItem>
+            {t('ceph-storage-plugin~External storage platform: {{storagePlatform}}', {
+              storagePlatform,
+            })}
+          </ListItem>
+        )}
+      </ReviewItem>
+      <ReviewItem title={t('ceph-storage-plugin~Capacity and nodes')}>
```

Comment:
can we consider nodes as CRD names here? is it Nodes or nodes?

Reply:
This is to match with navigation title. We also not capitalizing storage class if you check.

Reply:
I checked that from console and pf teams. Its correct.

Reply:
ack